pkg/common/peertracker/npipe_windows.go

```diff
@@ -55,7 +57,7 @@ func getNamedPipeClientProcessID(pipe windows.Handle, clientProcessID *int32) (e
 // compareObjectHandles compares two object handles to determine if they
 // refer to the same underlying kernel object
 func compareObjectHandles(firstHandle, secondHandle windows.Handle) error {
-	r1, _, e1 := syscall.SyscallN(procCompareObjectHandles.Addr(), uintptr(firstHandle), uintptr(secondHandle))
+	r1, _, e1 := syscall.Syscall(procCompareObjectHandles.Addr(), 2, uintptr(firstHandle), uintptr(secondHandle), 0)
```

Comment:
Don't we still want to use SyscallN here?

Reply:
Oops, missed that, thanks!